### DISCUSSION TITLE
Question : permission to use in a project

### DISCUSSION THREAD

**Discussion by kuipou** [simeydotme/pokemon-cards-css#27] [General]
I would like to know if it would be possible to be use in a theme project

**Comment by simeydotme** [comment]
please read the license
https://en.wikipedia.org/wiki/GNU_General_Public_License#Derivations

essentially; 
- if you copy files, or large bits of code, then your project need to use the same license (open source)
- if you look at the code and re-write some of it for your own purpose, that's up to you what you feel comfortable with doing. I'd call that "inspiration". but it is limited. I do not grant any additional usage rights to individuals.
